libpod/runtime.go

```diff
@@ -609,6 +585,14 @@ func makeRuntime(runtime *Runtime) (err error) {
 		SignaturePolicyPath: runtime.config.SignaturePolicyPath,
 	}
 
+	if rootless.IsRootless() {
+		rootlessRuntimeDir, err := util.GetRootlessRuntimeDir()
+		if err != nil {
+			return err
+		}
+		runtime.config.TmpDir = filepath.Join(rootlessRuntimeDir, fmt.Sprintf("%s-libpod", runtime.config.StorageConfig.GraphDriverName), "tmp")
```

Comment:
I think tests are failing because of this change, as we are always overriding `runtime.config.TmpDir` even if specified by the user.

I think it would be better to restore the getDefaultTmpDir mechanism